src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <section class="main-board">
+        <div id="display-score">
+            <div id="score-board">
+                <div id="reset-sec">
+                    <h3>Recent scores </h3>
+                    <button id="refresh-score-btn">Refresh</button>
+                </div>
+                <ul>
+                    <li>Name: 10</li>
+                    <li>Name: 100</li>
+                    <li>Name: 50</li>
+                    <li>Name: 25</li>
+                    <li>Name: 50</li>
+                    <li>Name: 25</li>
+                </ul>
+            </div>
+        </div>
+        <div id="set-score">
+            <form >
+                <h3>Add your score</h3>
+                <input type="text" placeholder="Your name">
+                <input type="text" placeholder="Your score">
+                <button type="submit" value="Submit" id="submit-score-btn">Submit</button>
+            </form>
+        </div>
+    </section>
+    <footer><p>© 2021 Leaderboard</p></footer>
```

Comment:
- [ ] One of the objectives of this milestone is to practice JS ES6 modules, functions, syntaxes, and DOM. I think the `plain HTML` phrase is used to emphasize that no styling using CSS is needed. Kindly construct and render components in JS DOM which are children of the root HTML element. :pray:

Reply:
Hi @henatan99 ,

Thank you for the feedback.
The DOM rendering has been done, but please I need more clarification:
Kindly elaborate on the part of the comment: Please do you mean I should remove the CSS styling completely?

W/Regards,
Michael

Reply:
No @micheaol , you shouldn't remove the CSS styling. I meant, it is not mandatory for this milestone. You should keep what you have already done.

Reply:
Thanks for the reply.